MessageCenter/Classes/Views/Chatting/GroupChannelChattingViewController.swift

```diff
@@ -622,7 +620,7 @@ class GroupChannelChattingViewController: UIViewController, SBDConnectionDelegat
         }
     }
     
-    @objc private func sendFileMessage() {
+    func openPicker() {
         let mediaUI = UIImagePickerController()
         mediaUI.sourceType = UIImagePickerControllerSourceType.photoLibrary
         let mediaTypes = [String(kUTTypeImage), String(kUTTypeMovie)]
```

Comment:
Only images should be selectable.
```suggestion
        let mediaTypes = [String(kUTTypeImage)]
```

Reply:
As far as I know, we will implement sending video file as well in the future.

Reply:
Yes, in the future.. the user shouldn't be able to select videos or see them in the picker now.